### PR TITLE
mgr/dashboard: fix drain e2e failure 

### PR DESCRIPTION
**Pull request by nizamial09**
Cypress sometimes fail to register the click and that causes the
deselect/select to not happen properly. Deselecting the row immediately
after performing the action makes it pass from cypress.

Fixes: https://tracker.ceph.com/issues/55741
Signed-off-by: Nizamudeen A <nia@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by nizamial09** [comment]
jenkins test dashboard

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
> Don't know that this `within()` change is doing, but I trust you ;-) What's the source of this change? Any other PR?

the unselect table looks to be the closest one I could find. anyway the qe failed again. so will have to look at it once more: https://pulpito.ceph.com/nizamial09-2022-06-01_11:42:39-rados:dashboard-wip-nia-testing-2022-06-01-1326-distro-default-smithi/

File: src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts [hunk 1]
```diff
@@ -568,33 +568,15 @@ describe('TableComponent', () => {
       expect(executingElement.nativeElement.textContent.trim()).toBe(`(${state})`);
     };
 
-    it('should display executing template', () => {
+    it.only('should display executing template', () => {
       testExecutingTemplate();
     });
 
-    it('should display executing template with custom classes', () => {
+    it.only('should display executing template with custom classes', () => {
```
Comment (Sarthak0702) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts:575]: @nizamial09 You can revert this back.

**Comment by nizamial09** [comment]
[QA run](https://pulpito.ceph.com/nizamial09-2022-06-09_18:25:44-rados:dashboard-wip-nia-testing-2022-06-09-2104-distro-default-smithi/) was successful

**Review by Sarthak0702 — APPROVED** [review]

**Comment by epuertat** [comment]
[e2e failure](https://jenkins.ceph.com/job/ceph-dashboard-pull-requests/10820/artifact/build/out/mgr.x.log) unrelated to this ([known issue](https://tracker.ceph.com/issues/53582)):
```
2022-06-13T08:39:31.029+0000 7f4ea534d100  1 mgr[py] Loading python module 'balancer'
2022-06-13T08:39:31.045+0000 7f4ea534d100 10 mgr[py] Computed sys.path '/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr:/usr/local/lib/python3.8/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.8/dist-packages:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/build/lib/cython_modules/lib.3:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/python-common::/usr/lib/python38.zip:/usr/lib/python3.8:/usr/lib/python3.8/lib-dynload'
2022-06-13T08:39:31.125+0000 7f4ea534d100 -1 mgr[py] Module not found: 'balancer'
2022-06-13T08:39:31.125+0000 7f4ea534d100 -1 mgr[py] Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/balancer/__init__.py", line 2, in <module>
    from .module import Module
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/balancer/module.py", line 229, in <module>
    class Module(MgrModule):
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/balancer/module.py", line 350, in Module
    def set_mode(self, mode: Mode) -> Tuple[int, str, str]:
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/mgr_module.py", line 416, in __call__
    self.store_func_metadata(func)
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/mgr_module.py", line 413, in store_func_metadata
    self._load_func_metadata(f)
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/mgr_module.py", line 397, in _load_func_metadata
    args.append(CephArgtype.to_argdesc(arg_spec[arg],
TypeError: to_argdesc() takes from 2 to 3 positional arguments but 4 were given

2022-06-13T08:39:31.125+0000 7f4ea534d100 -1 mgr[py] Class not found in module 'balancer'
2022-06-13T08:39:31.125+0000 7f4ea534d100 -1 mgr[py] Error loading module 'balancer': (2) No such file or directory
2022-06-13T08:39:31.125+0000 7f4ea534d100  1 mgr[py] Loading python module 'cephadm'
2022-06-13T08:39:31.145+0000 7f4ea534d100 10 mgr[py] Computed sys.path '/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr:/usr/local/lib/python3.8/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.8/dist-packages:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/build/lib/cython_modules/lib.3:/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/python-common::/usr/lib/python38.zip:/usr/lib/python3.8:/usr/lib/python3.8/lib-dynload'
2022-06-13T08:39:31.305+0000 7f4ea534d100 -1 mgr[py] Module not found: 'cephadm'
2022-06-13T08:39:31.305+0000 7f4ea534d100 -1 mgr[py] Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/cephadm/__init__.py", line 1, in <module>
    from .module import CephadmOrchestrator
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/cephadm/module.py", line 30, in <module>
    from cephadm.serve import CephadmServe
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/cephadm/serve.py", line 15, in <module>
    import orchestrator
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/orchestrator/__init__.py", line 3, in <module>
    from .module import OrchestratorCli
  File "/home/jenkins-build/build/workspace/ceph-dashboard-pull-requests/src/pybind/mgr/orchestrator/module.py", line 18, in <module>
    from ceph.deployment.drive_group import DriveGroupSpec, DeviceSelection, OSDMethod
ImportError: cannot import name 'OSDMethod' from 'ceph.deployment.drive_group' (/usr/lib/python3/dist-packages/ceph/deployment/drive_group.py)
```

**Pull request merged**